src/cpp/session/modules/SessionCodeTools.R

```diff
@@ -2622,3 +2622,15 @@
    # give up, i.e. use the user preference
    .rs.scalar("")
 })
+
+.rs.addJsonRpcHandler("cpp_is_cpp11_file", function(file)
```

Comment:
You might want to make this more generic, e.g. `cpp_required_package` (not sure what the best name is here), so that we could support Rcpp, cpp11, a theoretical future cpp17, and so on. (It would just make this easier to extend in the future if we ever needed to)

Reply:
I went for a `cpp_source_file` instead that makes the call and send it to the console.